fastapi_jsonapi/data_layers/sqla_orm.py

```diff
@@ -177,16 +273,55 @@ async def update_object(self, obj: Any, data: dict, view_kwargs: dict) -> bool:
         :params view_kwargs: kwargs from the resource view.
         :return: True if object have changed else False.
         """
-        pass
+        new_data = data_update.attributes.dict()
+        has_updated = False
+
+        for field_name, new_value in new_data.items():
+            # TODO: handle attr not found
+            old_value = getattr(obj, field_name)
```

Comment:
Todo: get field alias (if present) and get attribute by alias (rarely used, but required)